docs/index.rst

```diff
@@ -3,6 +3,370 @@
+user has read access to the keytab file. Currently, the connector requires that the principal
+has the keytab path to be the same on all the hosts running the connector.
+
+Configuration Options
+~~~~~~~~~~~~~~~~~~~~~
+``flush.size``
+  The number of records needed to rotate files.
+
+  * Type: int
+  * Default:
+  * Importance: high
+
+``hdfs.url``
+  HDFS connection URL. This configuration has the format of hdfs:://hostname:port and specifies
+  which HDFS to export data to.
+
+  * Type: string
+  * Default: ""
+  * Importance: high
+
+``connect.hdfs.keytab``
+  The path to the keytab file for the HDFS connector principal. This keytab file should only be
+  readable by the connector user
+
+  * Type: string
+  * Default: ""
+  * Importance: high
+
+``connect.hdfs.principal``
+  The principal to use when HDFS is using Kerberos to for authentication.
+
+  * Type: string
+  * Default: ""
+  * Importance: high
+
+``format.class``
+  The format class to use when writing data to HDFS.
+
+  * Type: string
+  * Default: "io.confluent.connect.hdfs.avro.AvroFormat"
+  * Importance: high
+
+``hadoop.conf.dir``
+  The Hadoop configuration directory.
+
+  * Type: string
+  * Default: ""
+  * Importance: high
+
+``hadoop.home``
+  The Hadoop home directory.
+
+  * Type: string
+  * Default: ""
+  * Importance: high
+
+``hdfs.authentication.kerberos``
+  Configuration indicating whether HDFS is using Kerberos for authentication.
+
+  * Type: boolean
+  * Default: false
+  * Importance: high
+
+``hdfs.namenode.principal``
+  The principal for HDFS Namenode.
+
+  * Type: string
+  * Default: ""
+  * Importance: high
+
+``hive.conf.dir``
+  Hive configuration directory
+
+  * Type: string
+  * Default: ""
+  * Importance: high
+
+``hive.home``
+  Hive home directory
+
+  * Type: string
+  * Default: ""
+  * Importance: high
+
+``hive.integration``
+  Configuration indicating whether to integrate with Hive when running the connector.
+
+  * Type: boolean
+  * Default: false
+  * Importance: high
+
+``hive.metastore.uris``
+  The Hive metastore URIs, can be IP address or fully-qualified domain name and port of the
+  metastore host
+
+  * Type: string
+  * Default: ""
+  * Importance: high
+
+``logs.dir``
+  Top level HDFS directory to store the write ahead logs.
+
+  * Type: string
+  * Default: "logs"
+  * Importance: high
+
+``partitioner.class``
+  The partitioner to use when writing data to HDFS. You can use DefaultPartitioner, which preserves
+  the Kafka partitions; FieldPartitioner, which partitions the data to different directories
+  according to the value of the partitioning field specified in partition.field.name configuration;
+  TimebasedPartitioner, which partitions the data according to the time ingested to HDFS.
+
+  * Type: string
+  * Default: "io.confluent.connect.hdfs.partitioner.DefaultPartitioner"
+  * Importance: high
+
+``retry.backoff.ms``
```

Comment:
This has importance *high*? Do we actually expect users to use this? What will they use it for?